source/users-groups-permissions.rst

```diff
@@ -4,12 +4,14 @@
 Lesson 4: Users, Groups, Permissions
```

Comment:
At some point, the files `/etc/shadow` and `/etc/gshadow` should be mentioned, at least in passing (maybe around line 170 or so)